server.go

```diff
@@ -393,6 +399,7 @@ func CreateTeam(w http.ResponseWriter, r *http.Request) {
 	} else {
 		w.WriteHeader(http.StatusNotModified)
 		log.Printf("Team [%s] not created (already exists).", team.Name)
+		// Comment
```

Comment:
Could this error be related to other than `already exists`?

Reply:
Since the Insert(*Team) function returns false if the team is already in the map and true if the team was added here the interpretation is I believe correct.